src/runtime/vectorize/server/utils/vectorize.ts

```diff
@@ -0,0 +1,107 @@
+import { ofetch } from 'ofetch'
+import { joinURL } from 'ufo'
+import { createError } from 'h3'
+import type { H3Error } from 'h3'
+import type { Vectorize } from '../../../../types/vectorize'
+import { requireNuxtHubFeature } from '../../../utils/features'
+import { useRuntimeConfig } from '#imports'
+
+let _vectorize: Vectorize
+
+/**
+ * Access the Vectorize database.
+ *
+ * @example ```ts
+ * const vectorize = hubVectorize()
+ * let vectorsToInsert = [
+ *   {id: "123", values: [32.4, 6.5, 11.2, 10.3, 87.9]},
+ *   {id: "456", values: [2.5, 7.8, 9.1, 76.9, 8.5]},
+ * ]
+ * let inserted = await vectorize.insert(vectorsToInsert)
+ * ```
+ *
+ * Vectorize is currently only supported with `--remote`. See https://developers.cloudflare.com/workers/testing/local-development/#supported-resource-bindings-in-different-environments for details.
```

Comment:
Maybe better to mention on docs, and link the user there rather than straight to Cloudflare docs

Reply:
Fyi this warning was added to the docs, but without linking to Cloudflare's docs. I think I'll just remove the link because Cloudflare adding support in local via getPlatformProxy doesn't automatically mean we support it without needing extra logic.